### PR TITLE
Changelog for v0.17.0-rc1

### PR DESCRIPTION
**Pull request by mooskagh**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+* --sticky-checkmates flag (default off), supposed to find shorter checkmate
+  sequences.
+* More features in backend "check".
+
+
+Performance optimizations:
+* Added --out-of-order-eval flag (default is off).
+  Switching it on makes cached/terminal nodes higher priority, which increases
+  nps.
+* OpenCL backend now supports batches (~5x speedup!)
```
Comment (borg323) [inline changelog.txt]: I think this is up to 5x

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+  during play. (For training games, it's 7)
+
+
+Removed features:
+* Backpropagation beta / backpropagation gamma parameters have been removed.
+
+
+Other changes:
+* Release .zip package now contains NVdia CUDA and cuDNN .dlls.
+
```
Comment (borg323) [inline changelog.txt]: * Release windows executables are built with "whole program optimization".

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+  This is fixed, but default is temporary changed to --cache-history-length=1
+  during play. (For training games, it's 7)
+
+
+Removed features:
+* Backpropagation beta / backpropagation gamma parameters have been removed.
+
+
+Other changes:
+* Release .zip package now contains NVdia CUDA and cuDNN .dlls.
```
Comment (borg323) [inline changelog.txt]: Mention full lc0-windows-cuda.zip name?

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,65 @@
...
+  probable) that best move decision could be changed if allowed to think more.
+* Lc0 now supports configuration files. Options can be listed there instead of
+  command line flags / uci params.
+  Config should be named lc0.config and located in the same directory as lc0.
+  Should list one command line option per line, with '--' in the beginning
+  being optional, for example:
+
+     syzygy-paths=/path/to/syzygy/
+
+* In uci info, depth is not average depth rather than full depth
```
Comment (fli) [inline changelog.txt]: I guess this should be `now average depth`

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,65 @@
+v0.17.0-rc1 (2018-08-19)
+~~~~~~~~~~~
+
+New visible features:
+* Implemented ponder support.
+* Tablebases are supported now (only WDL probe for now).
+  Command line parameter is
+  --syzygy-paths=/path/to/syzygy/
+* Old smart pruning flag is gone. Insted there is
```
Comment (fli) [inline changelog.txt]: typo for `instead`

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,65 @@
...
+  that saves considerable amount of RAM.
+
+
+Bugfixes:
+* Fixed en passant detection bug which caused the position after pawn moving by
+  two squares not counted towards threefold repetition even if en passant was
+  not possible.
+* Fixed the bug which caused --cache-history-length for values 2..7 work the
+  same as --cache-history-length=1.
+  This is fixed, but default is temporary changed to --cache-history-length=1
```
Comment (fli) [inline changelog.txt]: should be `temporarily` :)

File: changelog.txt [hunk 7]
```diff
@@ -48,7 +48,7 @@ Bugfixes:
   not possible.
 * Fixed the bug which caused --cache-history-length for values 2..7 work the
   same as --cache-history-length=1.
-  This is fixed, but default is temporary changed to --cache-history-length=1
+  This is fixed, but default is temporariry changed to --cache-history-length=1
```
Comment (dubslow) [inline changelog.txt]: Uh... another typo? should be "temporarily" I think

**Pull request merged**